woocommerce/woocommerce-gutenberg-products-block
# Store Customization > Wireframe and adjust width for the Just Arrived Full Hero pattern

## What

Make a few adjustments to the **Just Arrived Full Hero** pattern to ensure that:

- The content is wireframed
- The width is wide so that it can be a better fit for the templates within the Patterns Assembler
- The text for the button is moved to the patterns dictionary

Fixes #11114

## Why

These adjustments are required to match the designs for the templates within the Patterns Assembler.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new post
2. Insert the **Just Arrived Full Hero** pattern
3. Add an image to it and save
4. Make sure the styles match what is on the screenshots of this PR
5. Save the post and check the front end: make sure things are working as expected as well


## Screenshots or screencast



| Desktop | Mobile |
| ------ | ----- |
| <img width="1214" alt="Screenshot 2023-10-03 at 17 58 13" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/c928007f-ede0-419d-b528-e24ad50d228a"> |  <img width="385" alt="Screenshot 2023-10-03 at 18 00 10" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/2883b7d8-8b60-4c07-9baa-8c69ec5430fa"> |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Just Arrived Full Hero pattern: wireframe the content and adjust the pattern width.
